Mlem/Assets.xcassets/Default Community.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
We should add image assets as PDF, select single scale so it becomes universal. it saves app size